python_modules/dagster/dagster/_core/definitions/step_launcher.py

```diff
@@ -60,7 +60,7 @@ def __new__(
 
 _step_launcher_supersession = superseded(
     subject="StepLauncher",
-    additional_warn_text="Consider using Dagster Pipes instead. Learn more here: https://docs.dagster.io/concepts/dagster-pipes",
+    additional_warn_text="This API has been superseded, but there is no plan to remove this functionality. For new projects, we recommend using Dagster Pipes.",
```

Comment:
```suggestion
    additional_warn_text="This API has been superseded, but there is no plan to remove this functionality. For new projects, we recommend using Dagster Pipes. For more information, see https://docs.dagster.io/guides/build/external-pipelines/",
```

Reply:
I didn't realize we couldn't format links here, so added a link to the new Pipes docs